cltk/corpus/medieval/corpora.py

```diff
@@ -0,0 +1,9 @@
+MEDIEVAL_CORPORA=[
```

Comment:
Add `"""Medieval corpora available for download."""  
` in the top.

add credits or other information, for example `__author__ = 'author full name email>'
 `, Also you can add License. 
Could you take a look at cltk modules conventions.